scripts/build-joplin-pr.sh

```diff
@@ -0,0 +1,22 @@
+# Add your Joplin PR API and Media Links here...
+# - NOTE: make sure CMS_API has `/api/graphiql` endpoints
+# - AND... CMS_MEDIA has `/media` endpoints
+
+# Examples
+# - export CMS_API='https://joplin-pr-3244-guide-icon-tile.herokuapp.com/api/graphql'
+# - export CMS_MEDIA='https://joplin-pr-3244-guide-icon-tile.herokuapp.com/media'
+
+# Or,  maybe you want to use staging or production media
+# - Staging Media: `https://joplin-austin-gov-static.s3.amazonaws.com/staging/media`
+# - Production Media: `https://joplin-austin-gov-static.s3.amazonaws.com/production/media`
+
+export NODE_PATH='./src'
+
+export CMS_API='https://joplin-pr-3244-guide-icon-tile.herokuapp.com/api/graphql'
+export CMS_MEDIA='https://joplin-pr-3244-guide-icon-tile.herokuapp.com/media'
+
+yarn npm-run-all build-css build-js
+
+echo " 🏗 END of the Joplin PR Build 🏗 "
+
+http-server dist
```

Comment:
...for help testing locally when you need to test against a Joplin PR with API & Data
- This used to be` build-joplin-ie.sh`, but it's been proven to serve more general cases, so it's been updated for that.